src/irr/asset/CImageWriterTGA.cpp

```diff
@@ -159,7 +159,7 @@ bool CImageWriterTGA::writeAsset(io::IWriteFile* _file, const SAssetWriteParams&
 	
 	STGAExtensionArea extension;
 	extension.ExtensionSize = sizeof(extension);
-	extension.Gamma = format == asset::EF_R8_UNORM ? 1.0f : ((100.0f / 30.0f) - 1.0f);
+	extension.Gamma = (format == asset::EF_R8_UNORM) ? 1.0f : ((100.0f / 30.0f) - 1.0f);
```

Comment:
can you explain the second gamma choice? why does it work out to 2.33333 ?

IEC 61966-2-1 works out more closely to 2.23333

Reply:
Hmmmmm yes, I meant 2.23333. Typo, my fault.

Reply:
also, format==EF_R8_UNORM is not a check for gamma

Reply:
we have this beauty for this
https://github.com/buildaworldnet/IrrlichtBAW/blob/315172ea3d1b9b51da4c1ef30412c02b99f1066a/include/irr/asset/EFormat.h#L1085

Reply:
there should be a runtime counterpart to this, if not create one